tsconfig.build.json

```diff
@@ -46,7 +46,8 @@
       "path": "./packages/token-search-discovery-controller/tsconfig.build.json"
     },
     { "path": "./packages/transaction-controller/tsconfig.build.json" },
-    { "path": "./packages/user-operation-controller/tsconfig.build.json" }
+    { "path": "./packages/user-operation-controller/tsconfig.build.json" },
+    { "path": "./packages/app-metadata-controller/tsconfig.build.json" }
```

Comment:
Nit: It would be good to keep this list alphabetized (or at least as alphabetized as possible), can you move it up below `announcement-controller`?